docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -86,6 +98,9 @@ The ``security-server`` section is for configuring security server parameters
 * <SOFT_TOKEN_ID> default software token ID, normally 0 (zero).
 * <SOFT_TOKEN_PIN> should be substituted with a desired numeric pin code
 * <SECURITY_SERVER_FQDN_OR_IP> should be substituted with the IP address or host name of the installed security server, e.g. ss1
+* ``/path/to/signcert`` and ``/path/to/authcert`` should be given as paths referring to certificate locations,
```

Comment:
In order for the documentation style to be consistent with line 101 in the User Guide, please also update lines 86, 88 and 91 accordingly